packages/@okta/vuepress-theme-prose/assets/css/layouts/_darktheme.scss

```diff
@@ -0,0 +1,338 @@
+  }
+  .signup .signup--content {
+    .signup--form {
+      .goto-signin,
+      .splitter {
+        color: cv('darkgray');
+      }
+      .goto-signin a {
+        color: cv('signup');
+      }
+    }
+  }
+
+  .authorization-layout {
+    .row .field-wrapper,
+    .authorization-layout--content .auth0-banner {
+      color: cv('darkgray');
+    }
+  }
+
+  .oin--leader p a.magentaLink {
+    color: cv('signup');
+  }
+
+  .Button--redOutline, 
+  .Button--redOutline:active, 
+  .Button--redOutline:hover {
+    color: cv('white');
+    border-color: cv('white');
+  }
+
+  .company-logos--icon {
+    &:nth-child(1) {
+      background-image: none !important;
+      mask-repeat: no-repeat;
+      mask-position: center;
+      background-color: cv('white') !important;
+      -webkit-mask-image: url(/img/logos/baseball.svg);
+      mask-image: url(/img/logos/baseball.svg);
+    }
+    &:nth-child(2) {
+      background-image: none !important;
+      mask-repeat: no-repeat;
+      mask-position: center;
+      background-color: cv('white') !important;
+      -webkit-mask-image: url(/img/logos/splunk.svg);
+      mask-image: url(/img/logos/splunk.svg);
+    }
+    &:nth-child(3) {
+      background-image: none !important;
+      mask-repeat: no-repeat;
+      mask-position: center;
+      background-color: cv('white') !important;
+      -webkit-mask-image: url(/img/logos/adobe.svg);
+      mask-image: url(/img/logos/adobe.svg);
+    }
+    &:nth-child(4) {
+      background-image: none !important;
+      mask-repeat: no-repeat;
+      mask-position: center;
+      background-color: cv('white') !important;
+      -webkit-mask-image: url(/img/logos/jetblue.svg);
+      mask-image: url(/img/logos/jetblue.svg);
+    }
+    &:nth-child(5) {
+      background-image: none !important;
+      mask-repeat: no-repeat;
+      mask-position: center;
+      background-color: cv('white') !important;
+      -webkit-mask-image: url(/img/logos/experian.svg);
+      mask-image: url(/img/logos/experian.svg);
+    }
+  }
+  @media (max-width: 1023px) {
+    header.page-header {
+      .menu--slideout .header--links {
+        & > ul.menu--items {
+          &:not(.expanded) {
+            & > li {
+              span.link,
+              a.link {
+                color: cv('darkgray');
+              }
+              a.link.router-link-active {
+                color: cv('signup') !important;
+              }
+            }
+          }
+          li a.link:hover, 
+          li span.link:hover {
+            color: cv('signup');
+          }
+        }
+        & > ul.menu--items.expanded {
+          li.opened > span {
+            color: cv('signup');
+          }
+        }
+      }
+    }
+    .search--wrapper .SearchBox .CoveoOmnibox.magic-box .magic-box-input input::placeholder {
+      color: cv('text');
+    }
+  }
+  .pricing {
+    background-color: transparent;
+    .Button--redOutline {
+      color: cv('white');
+    }
+  }
+}
+
+// ADD CUSTOM WIDTH
```

Comment:
this comment doesn't describe the change. Maybe fix submenu width would be better